app/Http/Controllers/PlantController.php

```diff
@@ -295,7 +295,7 @@ public function destroy(Plant $plant)
      */
     public function transfer(Plant $plant, Request $request)
     {
-        $this->authorize('update', $plant);
+        $this->authorize('update', Plot::find($request->plot_id));
```

Comment:
You need both authorize methods here sonce the action affects both the plant and the plot.
```suggestion
        $this->authorize('update', $plant);
        $this->authorize('update', Plot::find($request->plot_id));
```

You also need to validate before authorizing the plot since `Plot::find(...)` can return null if the plot is not found or if the request doesn't contain plot_id.